tensorflow/python/training/optimizer.py

```diff
@@ -367,9 +367,7 @@ def compute_gradients(self, loss, var_list=None,
     if grad_loss is not None:
       self._assert_valid_dtypes([grad_loss])
     if var_list is None:
-      var_list = (
-          variables.trainable_variables() +
-          ops.get_collection(ops.GraphKeys.TRAINABLE_RESOURCE_VARIABLES))
+      var_list = variables.trainable_variables()
```

Comment:
@alextp So this line is safe to remove as your consideration is from community not the core?